### PR TITLE
feat(Firma con IO): [SFEQS-800] Add mixpanel FCI events

### PR DESCRIPTION
**Pull request by hevelius**
## Short description
This PR adds mixpanel events for FCI feature.

## List of changes proposed in this pull request
- Added analitycs events
- Added middleware to mixpanel
- Update screens
- Refactor of some function used into events data

### CODE REVIEW

**Comment by Vangaorth** [comment]
What do you think about wrapping and grouping every dedicated tracking call in a single file, under the `fci/analytics/` folder? In this way, anybody can see which FCI analytic events there are by looking into a single entry point (see `ts/utils/analytics.ts` for reference but keep in mind that such file will be split in the future)

File: ts/features/fci/components/SuccessComponent.tsx [hunk 1]
```diff
@@ -70,6 +73,7 @@ const SuccessComponent = (props: {
   switch (status) {
     case SignatureRequestStatusEnum.WAIT_FOR_SIGNATURE:
       dispatch(fciStartRequest());
+      trackFciDocOpening(expires_at, fciDocuments.length);
```
Comment (shadowsheep1) [inline ts/features/fci/components/SuccessComponent.tsx:76]: In every other cases you did `track` first and then `dispatch`. What's the best way to do this? I personally prefer tracking first and then dispatching. Anyway, I suggest to use always the same pattern. What do you think?

File: ts/features/fci/hooks/useFciAbortSignatureFlow.tsx [hunk 2]
```diff
@@ -44,6 +47,7 @@ export const useFciAbortSignatureFlow = () => {
       rightButton={{
         ...errorButtonProps(() => {
           dismiss();
+          trackFciUserExit(route.name);
           dispatch(fciEndRequest());
```
Comment (shadowsheep1) [inline ts/features/fci/hooks/useFciAbortSignatureFlow.tsx:51]: Same as above about "tracking" and "dispatching". What do you think to have a standard also for "dismissing" and "dispatching"?

File: ts/features/fci/hooks/useFciCheckService.tsx [hunk 3]
```diff
@@ -67,6 +68,7 @@ export const useFciCheckService = () => {
               })
             );
           }
+          trackFciUxConversion();
           dispatch(fciStartSigningRequest());
           dismiss();
```
Comment (shadowsheep1) [inline ts/features/fci/hooks/useFciCheckService.tsx:73]: Same as above about "tracking" and "dispatching". What do you think to have a standard also for "dismissing" and "dispatching"?

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]

File: ts/features/fci/components/SuccessComponent.tsx (hunk 1 above)
Comment (hevelius) [reply ts/features/fci/components/SuccessComponent.tsx:76]: Yep. It's a good point. Make sense to track first. Thx!

**Review by shadowsheep1 — APPROVED** [review]
LGTM

**Pull request merged**